SalesforceFoundation/HEDAP
# Fixing namespace error on HEDA Settings page when NPSP and HEDA are i…

…nstalled in the same org

The issue here was that a query to get the "hed" namespace was using a class name that is also in NPSP, meaning sometimes it returned "npsp", causing the error.

# Critical Changes

# Changes

- In the rare case that NPSP and HEDA are installed in the same org, the HEDA Settings page no longer loads with an error.

# Issues Closed

Fixes #581 

# New Metadata

# Deleted Metadata

# Testing Notes

This one is tricky to test:

1. Setup an org with the managed package version of NPSP, 
2. Next, install the HEDA managed package, including this change, to ensure that both namespaces are being used.
2. Navigate to the HEDA app, then HEDA Settings page.
3. The page should load correctly, with no "internal server error" message.
